src/components/encounter/encounter-form.component.tsx

```diff
@@ -42,7 +42,7 @@ interface EncounterFormProps {
   formSessionDate: Date;
   provider: string;
   location: SessionLocation;
-  visit?: Visit;
+  visit: Visit;
```

Comment:
Are we now making the visit mandatory?

Reply:
Yes, since it is now not being passed as a prop but rather being fetched within the root component, it wouldn't make sense to have it as optional anymore yet it is always available.